src/commands.cpp

```diff
@@ -1286,6 +1286,10 @@ void CommandLogin::procresult()
                     client->sessionkey.assign((const char *)sek, sizeof(sek));
                 }
 
+                // retrieve signing and chat keys to force their creation if doesn't exist
+                client->getua(client->finduser(me,1), "+puEd255");
```

Comment:
Would be good to abstract this to a generic `getUserAttribute` function that handles en/decryption transparently for private attributes as well as public attributes